<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EthernetTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/res/client_60.241610738255px_1153834_easyicon.net.png</normaloff>:/res/client_60.241610738255px_1153834_easyicon.net.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="6">
     <widget class="QComboBox" name="comboBox_prefix">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="pushButton_loadFile">
      <property name="text">
       <string>加载通讯文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器IP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>前缀</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_serverIP"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_serverPort_send"/>
    </item>
    <item row="1" column="6">
     <widget class="QComboBox" name="comboBox_suffix">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>\n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>\r\n</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QComboBox" name="comboBox_server_client">
      <item>
       <property name="text">
        <string>服务器</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>客户端</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="12">
     <widget class="QPushButton" name="pushButton_clear">
      <property name="text">
       <string>清空接收</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_serverIP_send"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>接收端口</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="8">
     <widget class="QLineEdit" name="lineEdit_input"/>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="pushButton_creat">
      <property name="text">
       <string>创建连接</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>后缀</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="13">
     <widget class="QTextBrowser" name="textBrowser_show_msg"/>
    </item>
    <item row="3" column="12">
     <widget class="QCheckBox" name="checkBox_pauseShow">
      <property name="text">
       <string>暂停显示</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QCheckBox" name="checkBox_saveLog">
      <property name="text">
       <string>保存log</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>接收IP</string>
      </property>
     </widget>
    </item>
    <item row="7" column="12">
     <widget class="QPushButton" name="pushButton_send">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QPushButton" name="pushButton_delete">
      <property name="text">
       <string>删除连接</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_serverPort"/>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="pushButton_timer">
      <property name="text">
       <string>加载定时文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfor">
    <property name="title">
     <string>Infor</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuInfor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/information_32px_1206695_easyicon.net.png</normaloff>:/res/information_32px_1206695_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_serverIP</tabstop>
  <tabstop>lineEdit_serverPort</tabstop>
  <tabstop>comboBox_prefix</tabstop>
  <tabstop>comboBox_suffix</tabstop>
  <tabstop>comboBox_server_client</tabstop>
  <tabstop>pushButton_creat</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_loadFile</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>lineEdit_serverIP_send</tabstop>
  <tabstop>lineEdit_serverPort_send</tabstop>
  <tabstop>lineEdit_input</tabstop>
  <tabstop>pushButton_send</tabstop>
  <tabstop>textBrowser_show_msg</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
